plugins/Marketplace/vue/src/Marketplace/Marketplace.vue

```diff
@@ -217,7 +220,10 @@ export default defineComponent({
         sort: event,
       });
     },
-    fetchPlugins() {
+    updateMarketplace() {
+      this.fetchPlugins(this.$emit('triggerUpdate'));
```

Comment:
```suggestion
      this.fetchPlugins(() => this.$emit('triggerUpdate'));
```

Otherwise the event will be emitted before `fetchPlugins()` is called.